bpfd/src/utils.rs

```diff
@@ -63,25 +61,18 @@ pub(crate) fn get_ifindex(iface: &str) -> Result<u32, BpfdError> {
 }
 
 pub(crate) async fn set_file_permissions(path: &str, mode: u32) {
```

Comment:
You can now use this util here as well -> https://github.com/bpfd-dev/bpfd/blob/main/bpfd/src/serve.rs#L152